.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": false
+}
```

Comment:
```suggestion
{
  "semi": false,
  "singleQuote": true,
  "printWidth": 100,
  "trailingComma": "none"
}
```

Reply:
Appreciate you adding this, it was needed. The suggested changes aligns this with our internal repos.